tests/util.py

```diff
@@ -14,8 +14,8 @@ def clean_xml(xml_str):
     Remove whitespace from XML.
     """
     parser = etree.XMLParser(remove_blank_text=True)
-    return etree.tostring(
-        etree.XML(xml_str, parser=parser))
+    return str(etree.tostring(
```

Comment:
Is `encoding="UTF-8"` not feasible here? You can't just use `str` to do this conversion IIRC, for example in Python 3:

    >>> str(b'b')
    "b'b'"
    >>> b'b'.decode('utf-8')
    'b'